Command/RunCommand.php

```diff
@@ -144,10 +144,12 @@ protected function execute(InputInterface $input, OutputInterface $output)
         set_time_limit($processTimeout); // Set timeout on the process
 
         if ($jobId = $input->getOption('id')) {
-            return $this->runLoop->runJobById($start, $jobId); // Run a single job
+            $this->runLoop->runJobById($start, $jobId); // Run a single job
+            return $this::SUCCESS;
```

Comment:
returning success always ignores the return value of runJobById. I recommend either pushing down the "SUCCESS" return into runJobById, or actually processing the result from runJobById.

Reply:
Looking at runJobById, it seems like it never returns anything other than 0. So one option would be to just change the return signature of that function to void.

Reply:
> Looking at runJobById, it seems like it never returns anything other than 0. So one option would be to just change the return signature of that function to void.

I did take the return value of runJobById always returning 0 into consideration when writing that return, I just never considered changing the return type. I've gone ahead and pushed that change up